calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -43,8 +43,6 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 
 **Not supported**
 - GKE
-- RKE
-- RKE2
```

Comment:
RKE and RKE2 is deprecated in CC. I think we should leave this as it is

Reply:
If the are deprecated, we don't want mention of them in the docs at all.